core/genesis.go

```diff
@@ -307,7 +307,7 @@ func (g *Genesis) toBlock(db ethdb.Database, triedb *trie.Database) *types.Block
 			panic(fmt.Sprintf("unable to commit genesis block: %v", err))
 		}
 	}
-	return types.NewBlock(head, nil, nil, nil, trie.NewStackTrie(nil), nil, false)
```

Comment:
I see places that were `nil, false` and places that were `nil, true` and neither place calls `WithExtData`. Is `recalc` unused when `nil` is provided? It seems like in places where `nil, true` is passed, we should be setting `header.ExtDataHash = EmptyExtDataHash` right?

Reply:
I guess this only happens in tests

Reply:
Yep, that's only in tests so I don't think it matters.